debug/index.html

```diff
@@ -69,12 +69,10 @@
     style: 'mapbox://styles/mapbox/streets-v8'
   });
 
-  map.addControl(new (mapboxgl.NavigationControl || mapboxgl.Navigation)({
-    position: 'top-left'
-  }));
+  map.addControl(new (mapboxgl.NavigationControl || mapboxgl.Navigation)(), 'top-left');
```

Comment:
Since this is a breaking change anyway, why not just refer to NavigationControl?